docs/source/whats-new.rst

```diff
@@ -38,6 +38,17 @@ Documentation
 Internal Changes
 ~~~~~~~~~~~~~~~~
 
+.. _whats-new.v0.0.13:
+
+v0.0.12 (03/16/2023)
+--------------------
+
+New Features
+~~~~~~~~~~~~
+
+- Added a :py:func:`DataTree.move`
+  By `JosiasAurel <ndjosiasaurel@gmail.com>`_.
+
```

Comment:
It doesn't look like this has been added in the correct place. It should be under the unreleased version v0.0.13